### PR TITLE
Render recently viewed items from the database

### PR DESCRIPTION
**Pull request by jessicasm**
These changes render items from the database instead of using dummy data. On each refresh, the items will be pulled from the database at random. Please let me know if you have any feedback. Thank you!

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const PORT = 3001;
 app.use(express.static(path.join(__dirname, '../client/dist')));
 app.use(bodyParser.json());
 
-app.get('/recentlyViewedItems', (req, res) => {
+app.get('/recentlyviewed', (req, res) => {
```
Comment (pxshelly) [inline server/index.js]: I would consider camelCase for this route.

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -3,17 +3,29 @@ import RecentlyViewed from './RecentlyViewed.jsx';
...
     super(props);
 
     this.state = {
-      items: dummyData // array of objects
+      items: [] // array of objects
     }
   }
 
+  componentDidMount() {
+    getRecentItems((err, data) => {
```
Comment (pxshelly) [inline client/src/components/App.jsx]: Although the AJAX request works, I believe that defining the request in componentDidMount directly will eliminate the need to export and import the AJAX request file. As a result, you will reduce the lines of code needed to be written.

**Review by pxshelly — APPROVED** [review]
Looks good!

File: server/index.js (hunk 1 above)
Comment (jessicasm) [reply server/index.js]: I ended up using hyphens for readability per REST API's naming guide.

File: client/src/components/App.jsx (hunk 2 above)
Comment (jessicasm) [reply client/src/components/App.jsx]: Good point! I refactored the code based off your feedback. Thank you!

**Pull request merged**